sdk/java/src/main/java/com/gojek/feast/FeastClient.java

```diff
@@ -47,11 +50,23 @@
    *
    * @param host hostname or ip address of Feast serving GRPC server
    * @param port port number of Feast serving GRPC server
+   * @param tracer tracer instance to be used for tracing grpc calls to Feast
    * @return {@link FeastClient}
    */
-  public static FeastClient create(String host, int port) {
+  public static FeastClient create(String host, int port, Tracer tracer) {
```

Comment:
Shouldn't we have a method without a tracer as well?

Reply:
The original method wasn't removed: https://github.com/feast-dev/feast/pull/962/files#diff-091f65854936e6e5c42af98a3893a2dfR70